src/SprykerEco/Zed/Computop/ComputopConfig.php

```diff
@@ -17,6 +17,7 @@ class ComputopConfig extends AbstractBundleConfig
     public const OMS_STATUS_NEW = 'new';
     public const OMS_STATUS_INITIALIZED = 'init';
     public const OMS_STATUS_AUTHORIZED = 'authorized';
+    public const OMS_STATUS_AUTHORIZE_REQUEST = 'authorize request';
```

Comment:
```suggestion
    protected const OMS_STATUS_AUTHORIZE_REQUEST = 'authorize request';
```

please also move it down to keep constants sorting by access modifier